packages/ranges/src/elements/MultiThumbRange.js

```diff
@@ -13,14 +13,21 @@ import classNames from 'classnames';
 import styled from 'styled-components';
 import debounce from 'lodash.debounce';
 import { KEY_CODES } from '@zendeskgarden/react-selection';
-import { withTheme, isRtl, getDocument } from '@zendeskgarden/react-theming';
+import { retrieveTheme, withTheme, isRtl, getDocument } from '@zendeskgarden/react-theming';
 import RangeStyles from '@zendeskgarden/css-forms/dist/range.css';
 
+const SLIDER_COMPONENT_ID = 'ranges.multi_thumb_range.slider';
+const TRACK_COMPONENT_ID = 'ranges.multi_thumb_range.track';
+const RAIL_COMPONENT_ID = 'ranges.multi_thumb_range.rail';
+const THUMB_COMPONENT_ID = 'ranges.multi_thumb_range.thumb';
```

Comment:
Now my second question is this will now break the auto theming table id docs as it won't find these, I'll pull down this PR and poke around my script to get it working

Reply:
Resolved in  5dbd11b